packages/features/bookings/Booker/types.ts

```diff
@@ -64,6 +64,10 @@ export interface BookerProps {
    * otherwise, the default value is selected
    */
   duration?: number | null;
+  /**
+   * Configures the selectable options for a multiDuration event type.
+   */
+  durationConfig?: number[];
```

Comment:
Shall we name it `durationOptions` instead ? I find that word `config` implies a more complex configuration than just a set of possible duration values.